.github/workflows/image-release.yaml

```
@@ -68,8 +71,8 @@ jobs:
       run: |
         docker image tag kuberay/apiserver:${{ steps.vars.outputs.sha_short }} quay.io/kuberay/apiserver:${{ steps.vars.outputs.sha_short }};
         docker push quay.io/kuberay/apiserver:${{ steps.vars.outputs.sha_short }};
-        docker image tag kuberay/apiserver:${{ steps.vars.outputs.sha_short }} quay.io/kuberay/apiserver:${{ github.event.inputs.tag }};
-        docker push quay.io/kuberay/apiserver:${{ github.event.inputs.tag }}
+        docker image tag kuberay/apiserver:${{ steps.vars.outputs.sha_short }} quay.io/kuberay/apiserver:${{ steps.tag.outputs.tag }};
+        docker push quay.io/kuberay/apiserver:${{ steps.tag.outputs.tag }}
```

Shouldn't we read the environment variable directly? Something like:

```suggestion
        docker push quay.io/kuberay/apiserver:"${tag}"
```